### ISSUE TITLE
refactor: Sayfa içerisindeki datanın belirli süre aralıklarıyla yenilenme özelliğinin iptal edilmesi hedeflenmektedir.

### ISSUE THREAD

**Issue by khnbzkrt** [acikkaynak/deprem-yardim-frontend#615]
### Refactor İsteği

** discord kullanıcı adı: Hakan Angın#9836 **

### Refactor isteğiniz bir sorunla ilgili mi? Lütfen açıklayın.

Mevcut durumda "Alanı Tara" butonu ile sayfa içerisindeki datayı güncelleyebiliyoruz. Aynı zamanda belirli zaman aralıklarıyla saya içerisindeki data otomatik olarak güncelleniyor. Ancak bu otomatik güncelleme işlemi hem işlevsiz bir geliştirme özelliği taşıyor hem de initial value değerinin 30sn sonraki sayaç başlangıcının ise 900 sn olması bir bug olarak algılanıyor. 900 sn lik değerin yüksek olması da kafa karışıklığına sebep olup farklı bir bug konusu haline geliyor. Bunun standardı oturtulamayacağı için ve sağladığı fayda göz önünde bulundurularak bu özelliğin kaldırılması planlanmaktadır.

### Ek Bağlam
![image](https://user-images.githubusercontent.com/67371416/217949658-6232e49b-d1a4-40f9-b24d-6559dca5dded.png)

**Comment by eraygundogmus** [comment]
@usirin karar verebilir misin? daha sonrasında labelları güncellemeni rica ediyorum

**Comment by keislamoglu** [comment]
Şu anda artış formülü `x *= x` şeklinde. İlk bekleme süresi **30** saniye olduğu için **30**, **30*30**, **900*900**, şeklinde gidiyor. Birden bire süre oldukça artıyor ve neredeyse ikinci requestten sonra bekleme süresi anlamsızlaşıyor. Burada artışı daha iyi formülize ederek optimum bir çözüme gidilebilir. Mesela artışın `x *= 3` şeklinde düzenlenmesiyle **30**, **30*3**, **90*3**, **270*3** gibi daha anlamlı bir sonuç alınabilir.

**Comment by keislamoglu** [comment]
Bu özellik eklenirken hedeflenen, ekranı açık bir şekilde aynı bölgeyi inceleyen kişiler/kurumlar olabilir, dolayısıyla sürekli butona basmak durumunda olmasınlar idi.

**Comment by aerdemsenturk** [comment]
Aslında alanı tara özelliği başlarda boundingbox dışındaki noktaları hariç tutarak json boyutunu küçük tutmak içindi diye hatırlıyorum. Şimdi ise json boyutu zaten çok küçülmüş, noktaların tamamı da çağrılsa çok hızlı geliyor. 

Dolayısıyla en başta tüm noktalar tek seferde gelse, sürekli istek yapmaya da bu "Alanı tara" butonuna da hiç gerek kalmayacak.

<img width="1210" alt="Ekran Resmi 2023-02-10 01 09 23" src="https://user-images.githubusercontent.com/25268260/217967503-08a5a29a-9bed-4316-8e47-31a5e6bb56f3.png">

**Comment by keislamoglu** [comment]
@aerdemsenturk yine de sayfayı canlı tutmak için aralıklı request atılmasında fayda görüyorum. Aksi halde sayfayı refreshlemek gerekir güncel data için.

**Comment by aerdemsenturk** [comment]
Doğru. Ama yine de butonlar biraz boşa yer kaplıyor hale geliyor orada. 
Refresh işini sol tarftaki buton yapabilirdi. Sayacı ise saniye saniye bilmemize gerek yok diye düşünüyorum. Son güncelleme x sn önce. gibi kenarda yazabilirdi. Üst taraf biraz daha ferahlamış olurdu. Daha minimal bir çözüm ve aynı kapıya çıkıyor.
